.ci/script.sh

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env bash
 
 CI_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+# shellcheck source=/dev/null
 source "${CI_DIR}/common.sh"
 
 travis_retry make fetchthirdparty
 make all
 make testfront
-luajit $(which luacheck) --no-color -q {reader,setupkoenv,datastorage}.lua frontend plugins
+luajit "$(which luacheck)" --no-color -q {reader,setupkoenv,datastorage}.lua frontend plugins
+find . -type f -name '*.sh' -not -path "./base/*" -not -path "./luajit-rocks/*" -print0 | xargs --null shellcheck
```

Comment:
I was thinking about the same thing! Yes, makes total sense to not fun tests if static analysis is not even passing.